### ISSUE TITLE
unable to compile on 64bit linux

### ISSUE THREAD

**Issue by mrfricks** [mmatyas/GridWars#1]
hi,
hopefully you are still around?
i would really like a 64bit binary of this as i don't think one exists.
unfortunately it fails to compile for me on 64bit puppy linux with this error
```
[ 22%] Processing:colordefs.bmx
[ 23%] Processing:images.bmx
[ 23%] Processing:sound.bmx
[ 23%] Processing:utils.bmx
[ 24%] Processing:vectorfont.bmx
[ 24%] Processing:gridparttrail.bmx
Compile Error: Unable to find overload for setcolor(Float,Float,Float). Argument #1 is "Float" but declaration is "Int".
```
any idea how to fix?
or do you know of a 64bit builds done previously?

cheers
phil

**Comment by mmatyas** [comment]
Hi, it seems your compiler is running in strict mode (ie. without the `-w` option). Using `BlitzMax_linux_x64_0.129.3.45` with `-w` it gives me warnings like this:

```
Compile Warning: In call to Function SetColor:Int(red:Int,green:Int,blue:Int). Argument #1 is "Float" but declaration is "Int".
```

The issue is that SetColor expects Int values but the code calls it with Floats. This is a hard error with strict mode, but only a warning with `-w`. On the other hand, I do get a compile error later:

```
Compile Error: Expression of type 'Int' cannot be invoked.
[/hdd/terra1/Dev/GridWars/gridwars.bmx;2600;0]
```

Looking at the code, for some reason `PowerUp()` is no longer recognized as a valid class method, apparently. Something might have changed on the compiler side, so perhaps using an older (way older!) release might fix the issue.

**Comment by mrfricks** [comment]
hi, thanks for the reply!

i'll have a search for an older blitzmax and if that fails i'll email blitzmax to see if they can help.
it's a nice classic game and would be good to keep it alive.

cheers
phil

**Comment by mrfricks** [comment]
hi,  
well i've managed to get a working-ish 64bit binary using blitzmax version 0.93.3.21 :)
it's not quite right though as it's not using GL for some reason so isn't as smooth as my old 32bit binary.
i'll try some of the other blitzmax versions and see if that solves it.

FYI my old 32bit binary on startup says 
```
taumel's build 9.3 2006 on 4.1
```

**Comment by mmatyas** [comment]
I was just going through an old drive of mine, and also found an old BMX version I used, `BlitzMax_Linux.x64_v0.99.3.31` .

**Comment by mmatyas** [comment]
I think this go fixed by #3.
